static/scss/general_styles.scss

```diff
@@ -0,0 +1,45 @@
+// General Style Rules
+
+body {
+  background-color: $page-background;
+  font-family: 'Roboto', helvetica, arial, sans-serif !important;
+  line-height: 1.4em;
+  }
+
+
+ .mdl-card {
+    padding: 20px 24px 30px;
```

Comment:
Indentation should be two spaces per indent, no tabs. I can show you where to set this. Also this may be useful for Sublime, which will make it easy to clean the whitespace up afterwards: https://packagecontrol.io/packages/CleanCSS